scripts/file-finder.py

```
@@ -0,0 +1,52 @@
+import io
+import os
+from datetime import datetime
+import json
+from firecloud import fiss
+from firecloud.errors import FireCloudServerError
+import firecloud.api as fapi
+import numpy as np
+import pandas as pd
+
+# User-set variables
+BILLING_PROJECT_ID="biodata-catalyst"
+WORKSPACE="TSV-AFY"
+SUBDIRECTORY="\/case1\/"
+TABLE_NAME="CRAMs"
+# Make sure to escape the slashes in your SUBDIRECTORY variable. For instance, 
+# if your files are in a folder called "testfiles" you will need to enter it 
+# as "/\testfiles/\" or else Python will get angry.
+
+# Call FireCloud API
+try:
+    bucket = os.environ["WORKSPACE_BUCKET"]
+    response = fapi.list_entity_types(BILLING_PROJECT_ID, WORKSPACE)
+    if response.status_code != 200:
+        print("Error in Firecloud, check your billing project ID and the name of your workspace.")
```

This should raise an Exception here, otherwise this will continue to run code.